packages/@aws-cdk/aws-docdb/package.json

```diff
@@ -91,6 +91,11 @@
   "engines": {
     "node": ">= 10.13.0 <13 || >=13.7.0"
   },
+  "awslint": {
+    "exclude": [
+      "attribute-tag:@aws-cdk/aws-docdb.DatabaseSecret.secretName"
```

Comment:
Why is this needed?

We desperately need to require "motivation" in the exclude list!

Reply:
Because secretName isn't an attribute.